test/fetch-test-deps.sh

```diff
@@ -1,10 +1,46 @@
 #!/usr/bin/env bash
-
-set -e
+set -e  # stop the script when any command fails
 
 cd "$(dirname "$0")"
 
-case "$1" in
+# Parse arguments and provide usage message.
```

Comment:
This function does not do that.
```suggestion
# Parse arguments and provide usage message.
```

Reply:
This particular function indeed only provides the usage message. The *section* in which the function appears both parses arguments and provides a usage message. What convention should I use to annotate what a section of a script does, as opposed to an individual function? Does such a convention exist?

Reply:
There's no such convention because you don't need to annotate simple shell script functions, let alone whole sections. It's just an internal test script, not a many-file program that gets continuous feature development and needs complex explanation/organization.